src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/importorder/InputImportOrderStaticGroupsNegative.java

```diff
@@ -0,0 +1,18 @@
+//non-compiled with javac: contains specially crafted set of imports for testing
+package com.puppycrawl.tools.checkstyle.checks.imports.importorder;
+/**
+ * This test-input is intended to be checked using following configuration:
+ *
+ * staticGroups = {}
+ * option = top
+ * ordered = true
+ * separated = true
+ * sortStaticImportsAlphabetically = true
+ * useContainerOrderingForStatic = false
+ *
+ */
+import static org.testng.AssertJUnit.fail;
+import static java.lang.String.format;
```

Comment:
As we define exact config for whole input file , please add trailing comment here "// violation".
It will allow us to read Inputs and verify behavior without jumping to UT file for expectations.

Input fail should have all : config, input content, explicit/inlined violation markers.

Reply:
done